packages/server/src/insights/insights.service.ts

```diff
@@ -0,0 +1,38 @@
+import { Injectable } from '@nestjs/common';
+import { Connection } from 'typeorm';
+import { InsightInterface } from './insights';
+
+// interface generateAllInsightsParams {
+//   insights: InsightInterface<any>[];
+//   filters: any;  // TODO
+// }
+
+@Injectable()
+export class InsightsService {
+  constructor(private connection: Connection) {}
+
+  async generateOutput({ insight, filters }): Promise<any> {
```

Comment:
some purpose statements on the functions in this service would help a lot with maintainability

Reply:
Good idea.